use_cases/py_scripts/create_function_group_and_share.py

```diff
@@ -0,0 +1,60 @@
+import asyncio
```

Comment:
can this be moved to examples/create_function_group_and_share.py? Also if we could support sharing existing FG would be nice.